Makefile

```diff
@@ -5,7 +5,7 @@ PKG := github.com/genuinetools/$(NAME)
 CGO_ENABLED := 1
 
 # Set any default go build tags
-BUILDTAGS ?= seccomp osusergo
+BUILDTAGS ?= seccomp osusergo dfrunmount dfsecrets dfssh
```

Comment:
These tags should not be needed if you vendor the latest version of BuildKit

Reply:
After trying a bit, vendoring the latest (8.1.0) version of BuildKit requires much more effort as it impacts refactoring "pull" and "push" commands due to changes on sessions (e.g., https://github.com/moby/buildkit/pull/1551).

Wouldn't it be better if we open a dedicated issue/PR ?